tweakwcs/linearfit.py

```diff
@@ -10,8 +10,15 @@
 """
 import logging
 import numbers
+from  packaging.version import Version
```

Comment:
```suggestion
from packaging.version import Version
```

This should fix the pep8 error

Reply:
done